src/client/rewards/rewardsActions.js

```diff
@@ -0,0 +1,45 @@
+import { getAuthenticatedUserName } from '../reducers';
+
+export const SET_MATCH_BOT_RULE = '@rewards/SET_MATCH_BOT_RULE';
+export const SET_MATCH_BOT_RULE_START = "@rewards/SET_MATCH_BOT_RULE'_START";
+export const SET_MATCH_BOT_RULE_ERROR = "@rewards/SET_MATCH_BOT_RULE'_ERROR";
+export const SET_MATCH_BOT_RULE_SUCCESS = "@rewards/SET_MATCH_BOT_RULE'_SUCCESS";
```

Comment:
You can use 'createAsyncActionType' helper

Reply:
fixed